esmvalcore/preprocessor/_time.py

```diff
@@ -711,3 +754,73 @@ def timeseries_filter(cube, window, span,
                                weights=wgts)
 
     return cube
+
+
+def resample_hours(cube, hours):
```

Comment:
I'm thinking `interval` or `every_n_hours` would be a better variable name here instead of `hours`.

Reply:
Changed to intervak